mirdata/datasets/mtg_jamendo_autotagging_moodtheme.py

```diff
@@ -0,0 +1,340 @@
+"""
+import csv
+import json
+import os
+from typing import Optional, Tuple, BinaryIO
+
+import librosa
+import numpy as np
+from mirdata import download_utils, jams_utils, core, io
+
+BIBTEX = """@conference {bogdanov2019mtg,
+    author = "Bogdanov, Dmitry and Won, Minz and Tovstogan, Philip and Porter, Alastair and Serra, Xavier",
+    title = "The MTG-Jamendo Dataset for Automatic Music Tagging",
+    booktitle = "Machine Learning for Music Discovery Workshop, International Conference on Machine Learning (ICML 2019)",
+    year = "2019",
+    address = "Long Beach, CA, United States",
+    url = "http://hdl.handle.net/10230/42015"
+}"""
+INDEXES = {
+    "default": "1.0",
+    "test": "1.0",
+    "1.0": core.Index(filename="mtg_jamendo_autotagging_moodtheme_index_1.0.json"),
+}
+DOWNLOAD_INFO = """
+    The audio files can be downloaded following the path described in https://github.com/MTG/mtg-jamendo-dataset#downloading-the-data
+    
+    To download audio, unpack and validate all tar archives:
+    
+    .. code-block:: console
+
+          mkdir /path/to/download
+          python3 scripts/download/download.py --dataset autotagging_moodtheme --type audio /path/to/download --unpack --remove
+    
+    Later add the files to a folder into mir_datasets called audio/ with the following structure:
+        > mtg_jamendo_autotagging_moodtheme/
+            > audios/
+                > 00/
+                ...
+                > 99/
+"""
+REMOTES = {
+    "metadata": download_utils.RemoteFileMetadata(
+        filename="metadata.zip",
+        url="https://zenodo.org/record/3826813/files/data.zip?download=1",
+        checksum="039ce10f267f6e4e9f72837c76d72b2f",
+        destination_dir=".",
+    )
+}
+
+
+class Track(core.Track):
+    """MTG_jamendo_autotagging_moodtheme Track class
+
+    Args:
+        track_id (str): track id of the track (JAMENDO track id)
+
+    Attributes:
+        audio_path (str): Path to the audio file
+
+    Cached Properties:
+        artist_id (str): JAMENDO artist id
+        album_id (str): JAMENDO album id
+        duration (float): track duration
+        tags (str): autotagging moodtheme annotations
+
+    """
+
+    def __init__(
+        self,
+        track_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            track_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
+        self.track_id = track_id
+        self.audio_path = self.get_path("audio")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The track's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    @core.cached_property
+    def artist_id(self) -> str:
+        return self._metadata()["metadata"][self.track_id]["ARTIST_ID"]
```

Comment:
```suggestion
        return self._track_metadata.get("ARTIST_ID")
```

and the same for `album_id`, `duration` and `tags`